docs/workflow/blocks/db.rst

```diff
@@ -3,3 +3,66 @@ Local Database
 
 The local database block implements an indexed database in the browser storage. 
 The block implements a version of Dexie.js. 
+
+
+Default config
+--------------
+
+.. code-block:: json
+
+    {
+      type: 'db',
+      operation: 'add',
+      adapterName: 'UNKNOWN',
+      schema: 'none',
+      skipFirst: true,
+    }
+
+Supported properties
+--------------------
+- **operation** - the database operation to undertake (See below)
+- **adapterName**  - the name of the adapter
+- **schema** -  the name of the schema
+- **schemaGetter** - A JMESPath expression to resolve the schema name
+- **skipFirst** - skip the first call (defaults to true)
```

Comment:
Should probably say more about this. I suppose this is a workaround of some Angular behaviour?

Reply:
Most blocks have a skipfirst option. This stops any activity the first time a block is triggered - as it will not have any data passed to it.